packages/qgrid-ngx-plugins/src/lib/data-provider/cache-requested-pages.ts

```diff
@@ -0,0 +1,45 @@
+import { GridModel } from '@qgrid/ngx';
```

Comment:
file name should be cache-already-requested-page-strategy.ts

Reply:
done